espnet2/tts/gradtts/gradtts.py

```diff
@@ -0,0 +1,350 @@
+from espnet2.tts.abs_tts import AbsTTS
+from espnet2.tts.gradtts.utils import (
+    duration_loss,
+    fix_len_compatibility,
+    generate_path,
+    sequence_mask,
+)
+
+
+class LayerNorm(BaseModule):
```

Comment:
Is there any difference between this layernorm and the torch's implementation? It seems similar to me